articles/openshift/oshift-lifecycle.md

```diff
@@ -0,0 +1,50 @@
+---
+title: UKCloud for OpenShift lifecycle and patch management | UKCloud Ltd
+description: Provides an overview of how UKCloud manages the patching and lifecycle support of its OpenShift service
+services: openshift
+author: Steve Dixon
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenShift lifecycle and patch management
+toc_fullpath: openshift/oshift-lifecycle.md
+toc_mdlink: oshift-lifecycle.md
+---
+
+# UKCloud for OpenShift life cycle and patch management
+
+## Overview
+
+This article outlines the life cycle of UKCloud supported OpenShift releases along with UKCloud's approach to patching exisitng OpenShift environments in order for customers to effectively plan, deploy, and support their infrastructure. UKCloud publishes this life cycle in an effort to provide as much transparency as possible and may make exceptions from this document should the need arise.
+
+### Intended audience
+
+Administrators and operators of the UKCloud for OpenShift service.
+
+## Life cycle managament
+
+Red Hat's official versioning for their OpenShift Container Platform is _**X.Y**_ where _**X**_ indicates the major version and _**Y**_ indicates a minor version.
+
+Although Red Hat officially fully support each major _**X**_ version of the OpenShift Container Platform for five years, as a service provider this duration is too long for UKCloud to ensure the highest levels of security, performance and features to their customers. As such, UKCloud will officially support the current and previous two minor versions **_(Y-2)_**. As minor versions are released roughly every six months, this provides users with the assurance their environments will be fully support by UKCloud for a minimum of twelve months.
```

Comment:
Suggest the following:

Although Red Hat officially fully supports each major _**X**_ version of the OpenShift Container Platform for five years, as a service provider, this duration is too long for UKCloud to ensure the highest levels of security, performance and features. As such, UKCloud will officially support the current (_**X.Y**_) and previous two minor versions (_**X.(Y-1)**_ and _**X.(Y-2)**_. As minor versions are released roughly every six months, this provides the assurance your environments will be fully support by UKCloud for a minimum of twelve months.